test/test_no_regression.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+from walter import project
+import pandas,os
+from walter.data_access import get_data_dir
+from pathlib2 import Path
+import numpy.testing as np
+
+def test_same_result():
+    #assert : verify if the same wheat field have the same results for the same chosen parameters.
+
+    p = project.Project(name='same') # Create the simulation directory
+    params = p.csv_parameters('sim_scheme_test.csv')[0] # recovery of parameters
+    params.update(dict(nbj=325)) #Adding parameters to vary
```

Comment:
The default value for nbj in the sim_scheme_test.csv file is 325. There is no need to update this value. To avoid any confusion, we might as well suppress this line.